lib/routes/account.js

```diff
@@ -1911,8 +1982,199 @@ module.exports = function (
             function () {
               reply({})
             },
+            function (err) {
+              reply(err)
```

Comment:
No biggie, but this could just be `reply` directly, instead of wrapping it in another function, right?